module/VuFind/src/VuFind/RecordDriver/DefaultRecord.php

```diff
@@ -1816,4 +1816,21 @@ public function getCoordinateLabels()
     {
         return (array)($this->fields['long_lat_label'] ?? []);
     }
+
+    /**
+     * Get a field as an array
+     *
+     * @param string $field Field
+     *
+     * @return array
+     */
+    protected function getFieldAsArray(string $field): array
+    {
+        // Make sure to return only non-empty values and avoid casting since description can be a PropertyString too:
+        $value = $this->fields['description'] ?? '';
+        if ('' === $value) {
+            return [];
+        }
+        return is_array($value) ? $value : [$value];
```

Comment:
Any reason not to use the simpler `(array)$value` here instead?

Reply:
It was mentioned in the comment further above, but I added a new one to clarify.